lua/entities/gmod_wire_expression2/core/selfaware.lua

```diff
@@ -93,7 +93,7 @@ end)
 --[[******************************************************************************]]--
 -- Name functions
 
-
+local SHORT_SECOND = 1 - engine.TickInterval()
```

Comment:
Just put this in the timer.Create. No need to store it.

Reply:
The `TickInterval` call and subtraction need to be redone if that's the case, but if you say so.